versioning-policy.md

```diff
@@ -39,9 +39,12 @@ try to. Once they are marked "stable" they have to follow these guidelines.
 
 <h3>API compatibility</h3>
 
-An API is any public class or interface exposed in Spark that is not marked as "developer API" or 
-"experimental". Release A is API compatible with release B if code compiled against release A 
-_compiles cleanly_ against B. Currently, does not guarantee that a compiled application that is 
+An API is any public class or interface documented in Spark, see also
```

Comment:
"... documented in Spark. See also ..."
But what are people meant to see there - what annotation means "API"?
No need to explain it here in full, but is the previous text not still applicable? Developer/experimental is not an API?

Reply:
Let me actually remove the link, and make it simpler.